app/services/scrapers/ocso_warrants.rb

```diff
@@ -0,0 +1,23 @@
+module Scrapers
+  class OcsoWarrants < ApplicationService
+    def perform
+      warrants = OcsoScraper::Crawler.perform
+      warrants.each do |warrant|
```

Comment:
for each warrant creating or finding  a Warrant record and assigning it with values.